main.js

```diff
@@ -22,6 +22,7 @@ try {
 		process.exit(1);
 	}
 }
+var Bot;
```

Comment:
Do not use a local variable for a value that is always accessible through the config object.
(Variables should also not start with capitaziled letters, those indicate classes and types.)